src/components/Navigation.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import { NavLink } from 'react-router-dom'
+
+Navigation.propTypes = {
+  pages: PropTypes.arrayOf(
+    PropTypes.shape({ title: PropTypes.string, path: PropTypes.string })
+  ),
+}
+
+export default function Navigation({ pages }) {
+  console.log(pages)
+  return (
+    <Nav>
+      {pages.map(({ title, path }) => (
+        <StyledNavLink key={title} to={path}>
+          {title}
+        </StyledNavLink>
+      ))}
+    </Nav>
+  )
+}
+
+const Nav = styled.nav`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+`
+
+const StyledNavLink = styled(NavLink)`
+  padding: 12px;
+  border: none;
+  font-family: 'Roboto';
```

Comment:
are you using the font only here? Maybe you want to set the font in your GlobbalStyles. Also, did you import the font somewhere?